### DISCUSSION TITLE
Syscalls symbols

### DISCUSSION THREAD

**Discussion by Desperado17** [KDAB/hotspot#474] [Q&A]
Greetings,

I have a trace where I see a ?? for some symbols in hotspot. All other debug symbols are present and visible. I suspect that these symbols are supposed to be inside syscalls. Is there a way to verify this and show the syscalls?

Regards

**Comment by milianw** [comment]
Syscalls have symbols, so that's probably not it. Rather, it's either kernel internals (for which you need the `kallsyms` file), or symbols from some other library that is missing debug information. The summary page should tell you more on that matter.
